src/vector/feature.rs

```diff
@@ -55,6 +71,52 @@ impl<'a> Feature<'a> {
         }
         return &self.geometry;
     }
+
+    pub fn create(&self, lyr: &Layer) -> Result<()> {
+        let rv = unsafe { ogr::OGR_L_CreateFeature(lyr.gdal_object_ptr(), self.c_feature) };
+        if rv != ogr_enums::OGRErr::OGRERR_NONE {
+            return Err(ErrorKind::OgrError(rv, "OGR_L_CreateFeature").into());
+        }
+        Ok(())
+    }
+
+    pub fn set_field_string(&self, field_name: &str, value: &str){
+        let c_str_field_name = CString::new(field_name).unwrap();
```

Comment:
Alright for returning `Result` on these methods.
However is it possible here to not use `unwrap()` ?